waspc/data/Generator/templates/sdk/wasp/core/serialization.ts

```diff
@@ -41,3 +70,7 @@ interface SuperJSONArray extends Array<SuperJSONValue> {}
 interface SuperJSONObject {
   [key: string]: SuperJSONValue
 }
+
+
+export const serialize = SuperJSON.serialize.bind(SuperJSON)
+export const deserialize = SuperJSON.deserialize.bind(SuperJSON)
```

Comment:
If we must do this, I recommend an arrow function + `typeof` to ensure the type is preserved, something like:
```typescript
export const serialize: typeof SuperJSON.serialize = (...args) => SuperJSON.serialize(...args)
```
But I think there's a simpler solution:
1. Import `serialize` and `deserialize` functions. They're independent of the class, and SuperJSON already does the binding internally.
2. Reexport them.